lamindb/models/schema.py

```diff
@@ -578,6 +578,13 @@ def __init__(
                 self.optionals.set(optional_features)
                 return None
         self._slots: dict[str, Schema] = {}
+        # if both features and a schema are provided, we use a slot for a new schema of the features
+        if features and slots:
+            main_schema = Schema(features=features).save()
+            slot_name = f"__external_{name}__" if name else "__external__"
```

Comment:
Why does this slot inherit the `name` of the schema? Why not just `__external__`?